packages/ag-charts-enterprise/src/series/cone-funnel/coneFunnelModule.ts

```diff
@@ -13,17 +13,16 @@ export const ConeFunnelModule: _ModuleSupport.SeriesModule<'cone-funnel'> = {
     moduleFactory: (ctx) => new ConeFunnelSeries(ctx),
     solo: true,
     tooltipDefaults: { range: 'nearest' },
-    defaultAxes: [
-        {
-            type: _Theme.CARTESIAN_AXIS_TYPE.CATEGORY,
-            position: _Theme.POSITION.BOTTOM,
-        },
-        {
-            type: _Theme.CARTESIAN_AXIS_TYPE.NUMBER,
-            position: _Theme.POSITION.LEFT,
-        },
-    ],
-    swapDefaultAxesCondition: (series) => series?.direction !== 'vertical',
+    defaultAxes: (series) =>
```

Comment:
Nit: Could this not also use `swapAxisCondition`?

Reply:
This will almost definitely end up with more complicated configuration - like the funnel

Reply:
Updated to use the function from the funnel series